gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Table.java

```diff
@@ -193,15 +228,15 @@ Job copy(String destinationDataset, String destinationTable, BigQuery.JobOption.
 
   /**
    * Starts a BigQuery Job to copy the current table to the provided destination table. Returns the
-   * started {@link Job} object.
+   * started {@link Job} object. ddd
```

Comment:
It seemed something important to remind the user of. If you don't like it I'll remove it :)